charts/alcide-advisor-cronjob/v2.1.0/app-readme.md

```diff
@@ -0,0 +1,48 @@
+# Alcide Kubernetes Advisor
+
+![Alcide Advisor](https://codelab.alcide.io/images/card-frontpage/frontpage-alcide-advisor.png "Alcide Advisor")
```

Comment:
Remove this link. The UI displays a logo based on the icon specified in the Charts.yaml, so this link is making the chart's main page show 2 big logos and look cluttered.